sdks/java/io/kafka/src/main/java/org/apache/beam/sdk/io/kafka/KafkaIOUtils.java

```diff
@@ -142,4 +144,14 @@ void update(double quantity) {
       return avg;
     }
   }
+
+  static byte[] getOrderedCode(long offset) {
+    OrderedCode orderedCode = new OrderedCode();
+    orderedCode.writeNumIncreasing(offset);
```

Comment:
could see if this shows up on profiles, it would be cheaper to have some static method to to encode a long as just as fixed length bigendian since we don't need separator or inf that ordered code supports.

Reply:
Tried updating to cheaper approach.